core/controllers/collection_editor.py

```diff
@@ -66,7 +66,6 @@ def get(self, collection_id):
             'SHOW_COLLECTION_NAVIGATION_TAB_STATS': (
                 feconf.SHOW_COLLECTION_NAVIGATION_TAB_STATS),
             'TAG_REGEX': feconf.TAG_REGEX,
-            'title': collection.title,
```

Comment:
If we only need the `collection.id` (line 63) then can we directly use `collection_id` and avoid making calls to the datastore (line 59)?

Reply:
True okay